lib/ansible/cli/galaxy.py

```diff
@@ -402,8 +402,13 @@ def execute_install(self):
             # query the galaxy API for the role data
 
             if role.install_info is not None and not force:
```

Comment:
we might want to change the following to a `else force:` to keep people from changing existing roles by default